### PR TITLE
Feat: Add two methods for counting teams and calculating percentage o…

### PR DESCRIPTION
**Pull request by JCBeldo**
### Describe your changes
- Adds teams_count method
- Adds percentage_home_wins
- Update the spec for these two methods

### Checklist before requesting a review
- [x] I have performed a self-review of my code.
- [x] I have added thorough tests - unit and integrated.

### Number of teammates to review this PR before merging
- [x] one 
- [ ] two 
- [ ] EVERYONE PLS

### CODE REVIEW

File: lib/game_team.rb [hunk 1]
```diff
@@ -7,16 +7,15 @@ class GameTeam
...
   def initialize(row)
    @game_id = row[:game_id].to_i
    @team_id = row[:team_id].to_i
-   @home_or_away = row[:HoA] 
-   @result = row[:result] 
-   @head_coach = row[:head_coach] 
-   @goals = row[:goals].to_i 
-   @shots = row[:shots].to_i  
-   @tackles = row[:tackles].to_i 
+   @home_or_away = row[:hoa]
```
Comment (grace-joh) [inline lib/game_team.rb:14]: thank you for catching this bug!

File: lib/stat_tracker.rb [hunk 2]
```diff
@@ -33,7 +33,8 @@ def all_teams
 
   def all_game_teams
     @game_teams_data.map do |row|
-      game_team = GameTeam.new(row)
+      # require 'pry'; binding.pry
```
Comment (grace-joh) [inline lib/stat_tracker.rb]:
```suggestion
```

File: lib/stat_tracker.rb [hunk 3]
```diff
@@ -61,4 +61,26 @@ def games_by_season
...
+  #   end
+  #   away_games
+  # end
+  def count_of_teams
+    @team_data.count
+  end
+
+  def percentage_home_wins
+    team_wins = all_game_teams.select do |team|
+      # require 'pry'; binding.pry
```
Comment (grace-joh) [inline lib/stat_tracker.rb]:
```suggestion
```

File: spec/stat_tracker_spec.rb [hunk 4]
```diff
@@ -53,14 +53,26 @@
...
-      expect(@stat_tracker.all_game_teams.count).to eq(48)
+      expect(@stat_tracker.all_game_teams.count).to eq(14882)
     end
   end
 
   describe '#games_by_season' do
     it 'games by season' do
       expect(@stat_tracker.games_by_season).to be_a(Hash)
-      require 'pry'; binding.pry
+      # require 'pry'; binding.pry
```
Comment (grace-joh) [inline spec/stat_tracker_spec.rb]:
```suggestion
```

File: lib/game_team.rb (hunk 1 above)
Comment (JCBeldo) [reply lib/game_team.rb:14]: Thanks for troubleshooting.

**Comment by JCBeldo** [comment]
It should be updated fully now.

File: lib/Julianable.rb [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+    # find total games played in games class. take team_id.HoA(away).win
+    # team_id.HoA(away).win/ total_games played .round(2)
+  end
+  
+  def percentage_ties
+    # percetage of games that end in a tie
+    # find total games played in games class. find tie_games
+    # tie_games/ total_games .round(2)
+  end
+end
```
Comment (grace-joh) [inline lib/Julianable.rb]: Do we want to add this file if all of our assigned methods are going into StatTracker?

**Review by grace-joh — APPROVED** [review]

File: lib/Julianable.rb (hunk 5 above)
Comment (JCBeldo) [reply lib/Julianable.rb]: Yes, thanks for resolving this.  Late night pushes diminish critical thinking.

**Pull request merged**